generators.ml

```diff
@@ -38,7 +38,13 @@ let unsupported = [
     (* RMEM sentinels *)
     String.equal "STOP_FETCHING"; String.equal "THREAD_START";
     (* CSR are expected to be different *)
-    String.equal "CSR"
+    String.equal "CSR";
+    (* Not in cheriot (just because no supervisor mode?) *)
+    String.equal "SFENCE_VMA";
+    (* Already know hints have problems, fences appear uninteresting *)
```

Comment:
What problems are there with hint instructions?

Reply:
Just the metadata stripping issue.  I think I wrote hints in the plural only because there are a few different instructions involved.

Reply:
I only removed them to cover more ground in the tests, so I've reenabled them now.